### ISSUE TITLE
Instruments cutting out

### ISSUE THREAD

**Issue by Kermalis** [Kermalis/NDSMusicStudio#3]
Easy example to spot is BW2's SEQ_BGM_PW_FILMING within the first few seconds.

**Comment by Kermalis** [comment]
HGSS SEQ_GS_VS_TRAINER_KANTO is pretty bad too.
Could be related to #1 as well, but not confirmed.

**Comment by Kermalis** [comment]
Best example is probably MUS_LAST_BOSS in Bowser's Inside Story. The synths cut out drastically and the synths start right at the song's intro

**Comment by PlatinumLucario** [comment]
As I was testing out a build of NDS Music Studio with the latest commit, I tested out most of the tracks in Pokémon Mystery Dungeon: Blue Rescue Team for any issues with playback. And I found one:

BGM_M_DUNGEON_CHIISANAMORI_01 has an instrument (swav) that doesn't play fully. It's instrument number 10 (or 9, if you're counting from 1-9 instead of 0-9), this one cuts off halfway as the instrument is played.

So looks like the issue isn't completely fixed just yet
